scripts/build

```diff
@@ -98,21 +102,41 @@ if [ ${ARCH} = s390x ]; then
     export GOARCH="s390x"
 fi
```

Comment:
Does windows support arm, armv7 or s390x archs?

Reply:
Do you mean does Windows OS support ARM64? It does, but I only tested the bring up on X64 platform.